src/IPGenerate/IPGenerator.cpp

```diff
@@ -436,7 +437,8 @@ bool IPGenerator::Generate() {
         jsonF << "   \"build_name\": " << inst->OutputFile().filename() << ","
               << std::endl;
         jsonF << "   \"build\": true," << std::endl;
-        jsonF << "   \"json\": \"" << ip_config_file << "\"," << std::endl;
+        jsonF << "   \"json\": \"" << jasonfile.filename().string() << "\","
```

Comment:
Typo
```suggestion
        jsonF << "   \"json\": \"" << jsonfile.filename().string() << "\","
```

Reply:
Yeah that's a holdout from the original implementation, it's used throughout the function, should i rename it?

Reply:
Yes, it will help during the search for json.

Reply:
Fixed